larray/core/group.py

```diff
@@ -727,6 +727,18 @@ def __init__(self, key, name=None, axis=None):
         # the same name. See test_array.py:test_...
         self.axis = axis
 
+    @property
+    def name(self):
+        return self._name
+
+    @name.setter
+    def name(self, name):
+        if isinstance(name, (Group, ABCAxis)):
+            name = name.name
```

Comment:
This is wrong. It will break for unnamed groups: it will use None as the name, which is not what we want (we want to use the label). Might want to use _to_tick or something like that (needs to be checked though).

Reply:
That was unclear in the issue explanation.